### DISCUSSION TITLE
Kafka version upgrades within same operator version

### DISCUSSION THREAD

**Discussion by edgarkz** [strimzi/strimzi-kafka-operator#9579] [Q&A] [answered]
Hello,
Regarding kafka version upgrades - its unclear if jumping multiple versions of kafka is supported or each minor version should be upgraded first.
For example within same strimzi operator version:
3.0.0 -> 3.1.0 -> 3.2.0 
or
3.0.0 -> 3.2.0 

Can you please advise which above path is preferred way to upgrade kafka version.
Thank you

**Comment by scholzj** [comment, marked as answer]
You can skip Kafka versions and go directly from 3.0.0 to 3.2.0. (That said, most Strimzi versions support only 2 minor Kafka releases. So while nothing prevents you from skipping the Kafka versions, in most cases it will be coupled with operator upgrade as well)
